xds/src/main/java/io/grpc/xds/client/XdsResourceType.java

```diff
@@ -148,15 +148,25 @@ ValidatedResourceUpdate<T> parse(Args args, List<Any> resources) {
       Any resource = resources.get(i);
 
       Message unpackedMessage;
+      String wrappedResourceName = null;
```

Comment:
If you use `""`, then it is never null and you can just check `isEmpty()` below.

Reply:
Done